swmm_views/vw_swmm_dividers.sql

```diff
@@ -0,0 +1,46 @@
+DROP VIEW IF EXISTS qgep_swmm.vw_dividers;
+
+
+--------
+-- View for the swmm module class dividers
+-- 20190329 qgep code sprint SB, TP
+-- Question attribute Diverted Link: Name of link which receives the diverted flow. overflow > fk_overflow_to
+ 
+--------
+
+CREATE OR REPLACE VIEW qgep_swmm.vw_dividers AS
+
+SELECT
+	ma.obj_id as Name,
+	st_x(wn.situation_geometry) as X_coordinate,
+	st_y(wn.situation_geometry) as Y_coordinate,
+	ma.identifier as description,
+	concat(ma.remark, ',', wn.remark) as tag,
+	wn.bottom_level as invert_elev,
+	(co.level-wn.bottom_level) as max_depth,
+	'???' as diverted_link
+FROM qgep_od.vw_manhole as ma
+LEFT JOIN qgep_od.vw_wastewater_node wn on fk_wastewater_structure = ma.obj_id
```

Comment:
It will be easier to maintain if you join `od` tables instead of views.
Reasoning: on updates, views (like `vw_wastewater_node`) will be dropped and recreated. If the SWMM views depend on other views, they will need to be dropped recreated as well. And they will be more sensitive to changes on the underlying views.
So while it's a little more initial effort to join more tables now, it will save us maintenance trouble down the road.